packages/bot-web-ui/src/stores/load-modal-store.ts

```diff
@@ -318,6 +310,8 @@ export default class LoadModalStore implements ILoadModalStore {
             });
         }
         load({ block_string: this.selected_strategy.xml, drop_event: {}, workspace: this.recent_workspace });
+        const { updateBotName } = this.root_store.save_modal;
```

Comment:
maybe better use the syntax of ES6 with full destructuring assignment instead of the old approach? But that's just a suggestion, a taste of the code style🙂
```suggestion
        const { save_modal: { updateBotName }  } = this.root_store;
```

Reply:
done ✅